scripts/Dockerfile_Environment

```diff
@@ -25,13 +25,13 @@ RUN apt update && apt install -y git build-essential software-properties-common
 RUN curl -L https://foundry.paradigm.xyz | bash && . /root/.bashrc && foundryup
 
 # install solc
-RUN wget https://github.com/ethereum/solidity/releases/download/v0.8.20/solc-static-linux -O /usr/bin/solc && chmod +x /usr/bin/solc
+RUN wget https://github.com/ethereum/solidity/releases/download/v0.8.19/solc-static-linux -O /usr/bin/solc && chmod +x /usr/bin/solc
 
 # install go
-RUN wget https://go.dev/dl/go1.20.6.linux-amd64.tar.gz && rm -rf /usr/local/go && tar -C /usr/local -xzf go1.20.6.linux-amd64.tar.gz && echo 'PATH=$PATH:/usr/local/go/bin:/root/go/bin' >> ~/.bashrc
+RUN wget https://go.dev/dl/go1.19.linux-amd64.tar.gz && rm -rf /usr/local/go && tar -C /usr/local -xzf go1.19.linux-amd64.tar.gz && echo 'PATH=$PATH:/usr/local/go/bin:/root/go/bin' >> ~/.bashrc
```

Comment:
do you think this could have an effect on the orchestrator? should it use 1.19 as well?

Reply:
[question] out of curiosity why downgrade instead of upgrade? Is there an issue with Go 1.20.6 that prevents this repo from upgrading CI and this script to Go 1.20.6 (or even Go 1.21)?

Reply:
not at all. we can upgrade at any time. But I wanted to have the scripts match the exact environment as we have now, so that I am able to generate the right wrappers. Then, we can bump the versions in CI and in the dockerfile and shouldn't be an issue. I will open an issue for this

Reply:
Thanks for explaining! Agreed CI should match Dockerfile so this PR makes sense. When we bump Go, we should do it in all places for consistency.